pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -38,13 +35,14 @@ import (
 
 	// Loading registered providers.
 	_ "github.com/external-secrets/external-secrets/pkg/provider/register"
-	schema "github.com/external-secrets/external-secrets/pkg/provider/schema"
+	"github.com/external-secrets/external-secrets/pkg/provider/schema"
 	"github.com/external-secrets/external-secrets/pkg/template"
-	utils "github.com/external-secrets/external-secrets/pkg/utils"
+	"github.com/external-secrets/external-secrets/pkg/utils"
 )
 
 const (
-	requeueAfter = time.Second * 30
+	requeueAfter       = time.Second * 30
+	annotationDataHash = "eso-hash-map"
```

Comment:
I suggest naming this DNS-like under the `external-secrets.io` domain and define it in the `apis/externalsecret/v1alpha1` directory. 

Something like: `reconcile.external-secrets.io/data-hash`? Naming is hard :sweat_smile: